README.md

```diff
@@ -33,19 +33,20 @@ Each myNode device comes with a number of wonderful features:
 - Lightning Node (LND v0.9.0-beta)
 - Electrum Server (Electrum Rust Server 0.7.0)
 - BTCPay Server (v1.0.3.153)
-- Samourai Whirlpool
-- Mempool.Space local full node integration
-- JoinMarket CLI
+- RTL (Ride the Lightning) - Lightning Wallet (v0.6.5)
+- Samourai Whirlpool (whirlpool-client-cli v0.10.2)
+- Bitcoin RPC Explorer (v1.1.8)
+- Mempool.Space - Mempool local visualizer for the Bitcoin blockchain
+- LND Hub - Lightning Wallet Server (v1.1.3)
+- loopd - A Non-Custodial Off/On Chain Bridge (v0.4.0-beta)
+- Tor (v0.4.2.5)
```

Comment:
Tor is not at that rev for everyone, I reverted it back to the OS default which varies

Reply:
Ok, i removed the version